R/select_best.R

```diff
@@ -79,23 +79,9 @@ show_best.tune_results <- function(x, metric = NULL, n = 5, eval_time = NULL, ..
   metric_info <- choose_metric(x, metric)
   metric <- metric_info$metric
 
-  summary_res <- estimate_tune_results(x)
+  eval_time <- choose_eval_time(x, metric, eval_time = eval_time)
 
-  # TODO
```

Comment:
A fair amount of collateral refactoring happens here to more modularize the selection/filtering code.